code/game/objects/structures/window.dm

```diff
@@ -19,6 +19,32 @@
 	var/damageable = TRUE
 	var/deconstructable = TRUE
 
+
+/obj/structure/window/Initialize(mapload, start_dir, constructed)
+	..()
```

Comment:
```suggestion
	. = ..()
```

Reply:
This has `return INITIALIZE_HINT_LATELOAD` at the end.

Reply:
Muh consistency

Reply:
do the change